Gopkg.toml

```diff
@@ -94,7 +94,7 @@ required = [
   name = "github.com/giantswarm/micrologger"
 
 [[constraint]]
-  branch = "master"
+  branch = "e2e"
```

Comment:
Just for testing and before merging https://github.com/giantswarm/operatorkit/pull/232 because this was a bit bigger. Will sort this once I am confident about the changes and e2e tests are green.